symbols/arm9.yml

```
@@ -3138,12 +3146,30 @@ arm9:
         NA: 0x2012B34
         JP: 0x2012B04
       description: "Note: unverified, ported from Irdkwia's notes"
+    - name: GenerateCroagunkItems
+      address:
+        EU: 0x2012C24
+      description: |-
+        Generates the Croagunk Swap Shop items for Croagunk in Wigglytuff's Guild.
+        return: 1 if at least one item generated, 0 otherwise.
     - name: GetSynthItem
       address:
         EU: 0x20132F8
         NA: 0x2013250
         JP: 0x2013220
-      description: "Note: unverified, ported from Irdkwia's notes"
+      description: |-
+        Searches synth_template in SYNTH/synth.bin for a matching exclusive item.
+        r0: item_id of an exclusive item.
+        return: pointer to synth_template containing that item. 0 if no match was found.
+    - name: GetValidSynthsForSpecies
+      address:
+        EU: 0x201334C
+      description: |-
+        Fills out monster_synth_data for the specified monster, containing the synth templates that apply to said monster.
+        r0: monster_id of the monster
+        r1: Pointer to an empty monster_synth_data, to be filled by the function.
```

Per convention in some other places, let's do

```suggestion
        r1: [output] Pointer to an empty monster_synth_data, to be filled by the function.
```